tm2py/components/network/transit/transit_skim.py

```diff
@@ -330,26 +358,27 @@ def _calc_boardings(self, time_period: str, transit_class_name: str):
         TODO convert this type of calculation to numpy
         """
         _tp_tclass = f"{time_period}_{transit_class_name}"
-        spec = {
-            "type": "MATRIX_CALCULATION",
-            "constraint": {
-                "by_value": {
-                    "od_values": f'mf"{_tp_tclass}_XFERS"',
-                    "interval_min": 0,
-                    "interval_max": 9999999,
-                    "condition": "INCLUDE",
-                }
-            },
-            # CHECK should this be BOARDS or similar, not xfers?
-            "result": f'mf"{_tp_tclass}_XFERS"',
-            "expression": f'(mf"{_tp_tclass}_XFERS" - 1).max.0',
-        }
+        if ("PNR_TRN_WLK" in _tp_tclass) or ("WLK_TRN_PNR"in _tp_tclass):
```

Comment:
@arashasadabadi: check that the UEC changes are not reducing the number of boardings twice.